docker-compose.yml

```diff
@@ -13,11 +13,11 @@ services:
   vault:
     image: vault
     volumes:
-      - $PWD/example:/tmp/example
+      - $PWD/examples:/tmp/examples
```

Comment:
Why not keeping `example` as name for the folder?

Reply:
Because there are more than one example :) it's more descriptive

Reply:
Ok, fine with this change but the are still some inconsistencies e.g. https://github.com/kr1sp1n/node-vault/pull/78/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R23 and the comment in the first line of the example files.

Reply:
Oops. I forgot the package.json stuff. Yup I will fix that, and maybe also add test, lib and doc [directories](https://docs.npmjs.com/files/package.json#directories).

Reply:
@kr1sp1n Done